packages/common/src/search/_internal/hubEventsHelpers/processAttendeeFilters.ts

```diff
@@ -17,6 +17,11 @@ export function processAttendeeFilters(
   const useElseJoin = (value: string, defaults: string[]): string =>
     value?.length ? value : defaults.map((val) => val.toLowerCase()).join(",");
 
+  const userId = getOptionalPredicateStringsByKey(query.filters, "userId");
```

Comment:
Can we expand the comment for `hubSearchEventAttendees` to enumerate all the supported predicates similar to what we did for the `hubSearchEvents` method? This will require devs to do less reverse-engineering to identify the supported filters/predicates